### DISCUSSION TITLE
Let's earn the Galaxy Brain achievement

### DISCUSSION THREAD

**Discussion by eternitywen** [huiyadanli/RevokeMsgPatcher#456] [General]
Galaxy Brain | Bronze 🥉 --> 8 accepted answers

Galaxy Brain | Silver 🥈 --> 16 accepted answers

Galaxy Brain | Gold 🥇 --> 32 accepted answers

**Comment by xiaowoonly** [comment]
Well... Here is my first answer. If you accept it then I am closer.

**Comment by xiaowoonly** [comment]
And this is my second answer.
